api/src/main/java/org/pmiops/workbench/cohortreview/CohortReviewServiceImpl.java

```diff
@@ -108,13 +113,72 @@ public void saveFullCohortReview(CohortReview cohortReview, List<ParticipantCoho
 
     @Override
     public List<ParticipantCohortStatus> saveParticipantCohortStatuses(List<ParticipantCohortStatus> participantCohortStatuses) {
-        int i = 0;
-        for (ParticipantCohortStatus participantCohortStatus : participantCohortStatuses) {
-            entityManager.persist(participantCohortStatus);
-            i++;
-            if (i % batchSize == 0) {
-                entityManager.flush();
-                entityManager.clear();
+//        int i = 0;
+//        for (ParticipantCohortStatus participantCohortStatus : participantCohortStatuses) {
+//            entityManager.persist(participantCohortStatus);
+//            i++;
+//            if (i % batchSize == 0) {
+//                entityManager.flush();
+//                entityManager.clear();
+//            }
+//        }
+//        return participantCohortStatuses;
+        PreparedStatement preparedStatement = null;
+        Connection connection = null;
+        try {
+            connection = jdbcTemplate.getDataSource().getConnection();
+
+            connection.setAutoCommit(true);
```

Comment:
I thought this should be false?

Reply:
So I tried it with the auto commit set to false i and was running into strange errors with the database locking rows. I have only been able to successfully insert 10,000 rows locally with is set to true. Figured i would give it a try in the cloud and see what happens.